docs/policygenerator-reference.yaml

```
@@ -43,6 +43,10 @@ policyDefaults:
     # avoid evaluating the policy after it has become a particular compliance state.
     compliant: 30m
     noncompliant: 45s
+  # Optional. Determines whether objects created or monitored by the policy should be deleted when the policy is
+  # deleted. Pruning only takes place if the remediation action of the policy has been set to "enforce". Accepts values
```

So we don't have to update this doc every time a new option is supported in the API.
```suggestion
  # deleted. Pruning only takes place if the remediation action of the policy has been set to "enforce". Example values
```